client/components/PetTile.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+const PetTile = (props) => {
+  const { id, name, age, vaccinationStatus, adoptionStory, availableForAdoption, petTypeId, type, imgUrl } = props.pet
+  let isVaccinated = ""
+
+  if (vaccinationStatus) {
+    isVaccinated = "Yes"
+  } else {
+    isVaccinated = "No"
+  }
+
+  return (
+    <div class="pet-tile">
+      <div class="pet-pic">
+        <Link to={`/pets/${type}/${id}`}>
+          <img src={imgUrl} alt={`Picture of pet named ${name}`} />
+        </Link>
+      </div>
+      <div>
+        <Link to={`/pets/${type}/${id}`}>

```

Comment:
These 2 Links are the same, you can use one to wrap around everything you want to be clickable for the link